docs/UPP_GRIB2_Table_byID.csv

```diff
@@ -0,0 +1,868 @@
+Potential temperature on pressure surface,POT_ON_ISOBARIC_SFC,POT,14
+Dew point temperature on pressure surface,DPT_ON_ISOBARIC_SFC,DPT,15
+Specific humidity on pressure surface,SPFH_ON_ISOBARIC_SFC,SPFH,16
+Relative humidity on pressure surface,RH_ON_ISOBARIC_SFC,RH,17
+U-component wind on pressure surface,UGRD_ON_ISOBARIC_SFC,UGRD,18
+V-component wind on pressure surface,VGRD_ON_ISOBARIC_SFC,VGRD,19
+Omega on pressure surface,VVEL_ON_ISOBARIC_SFC,VVEL,20
+Absolute vorticity on pressure surface,ABSV_ON_ISOBARIC_SFC,ABSV,21
+Turbulent kinetic energy on pressure surface,TKE_ON_ISOBARIC_SFC,TKE,22
+Mesinger (Membrane) sea level pressure,MSLET_ON_MEAN_SEA_LVL,MSLET,23
+Surface pressure,PRES_ON_SURFACE,PRES,24
+Terrain height,HGT_ON_SURFACE,HGT,25
+Skin temperature,TMP_ON_SURFACE,TMP,26
+Skin potential temperature,POT_ON_SURFACE,POT,27
+Skin specific humidity,SPFH_ON_SURFACE,SPFH,28
+Skin dew point temperature,DPT_ON_SURFACE,DPT,29
+Lifted index—surface based,LFTX_ON_ISOBARIC_SFC_500-1000hpa,LFTX,30
+Lifted index—best,4LFTX_ON_SPEC_PRES_ABOVE_GRND,4LFTX,31
+Surface CAPE,CAPE_ON_SURFACE,CAPE,32
+Accumulated convective precipitation,ACM_ACPCP_ON_SURFACE,ACPCP,33
+Accumulated grid-scale precipitation,ACM_NCPCP_ON_SURFACE,NCPCP,34
+Accumulated snowfall,ACM_WEASD_ON_SURFACE,WEASD,35
+Total soil moisture,SOILM_ON_DEPTH_BEL_LAND_SFC,SOILM,36
+Low level cloud fraction,LCDC_ON_LOW_CLOUD_LYR,LCDC,37
+Mid level cloud fraction,MCDC_ON_MID_CLOUD_LYR,MCDC,38
+High level cloud fraction,HCDC_ON_HIGH_CLOUD_LYR,HCDC,39
+Temperature tendency from shortwave radiative flux,SWHR_ON_HYBRID_LVL,SWHR,40
+Temperature tendency from longwave radiative flux,LWHR_ON_HYBRID_LVL,LWHR,41
+Surface latent heat flux - time-averaged,AVE_LHTFL_ON_SURFACE,LHTFL,42
+Surface sensible heat flux - time-averaged,AVE_SHTFL_ON_SURFACE,SHTFL,43
+Roughness length,SFCR_ON_SURFACE,SFCR,44
+Friction velocity,FRICV_ON_SURFACE,FRICV,45
+Surface momentum flux - time-averaged,AVE_MFLX_ON_SURFACE,MFLX,46
+Accumulated surface evaporation,ACM_EVP_ON_SURFACE,EVP,47
+Latitude,NLAT_ON_SURFACE,NLAT,48
+Longitude,ELON_ON_SURFACE,ELON,49
+Land sea mask (land=1  sea=0),LAND_ON_SURFACE,LAND,50
+Sea ice mask,ICEC_ON_SURFACE,ICEC,51
+Press at tropopause,PRES_ON_TROPOPAUSE,PRES,54
+Temperature at tropopause,TMP_ON_TROPOPAUSE,TMP,55
+U wind at tropopause,UGRD_ON_TROPOPAUSE,UGRD,56
+V wind at tropopause,VGRD_ON_TROPOPAUSE,VGRD,57
+Wind shear at tropopause,VWSH_ON_TROPOPAUSE,VWSH,58
+Temperature at flight levels,TMP_ON_SPEC_ALT_ABOVE_MEAN_SEA_LVL,TMP,59
+U wind at flight levels,UGRD_ON_SPEC_ALT_ABOVE_MEAN_SEA_LVL,UGRD,60
+V wind at flight levels,VGRD_ON_SPEC_ALT_ABOVE_MEAN_SEA_LVL,VGRD,61
+Freezing level height (above mean sea level),HGT_ON_0C_ISOTHERM,HGT,62
+Freezing level RH,RH_ON_0C_ISOTHERM,RH,63
+10 M u-component wind,UGRD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m,UGRD,64
+10 M v-component wind,VGRD_ON_SPEC_HGT_LVL_ABOVE_GRND_10m,VGRD,65
+**Relative humidity on sigma levels 0.33-1.0**,RH_ON_SIGMA_LVL_0.33-1.0,RH,66
```

Comment:
@hertneky Just that these are my additions, and so they are most in need of checking. Doesn't matter as much on GitHub because it shows the diff, but if anyone prefers to view the docs on readthedocs, that's how they'd know what is new.

Reply:
@gspetro-NOAA Okay, I can look them over when I get a chance, likely not today.